.prettierrc.yml

```diff
@@ -0,0 +1,5 @@
+# style settings, see https://prettier.io/docs/en/options.html
+semi: false
```

Comment:
I'd vote we just keep the default prettier config. For a community project I think we should keep semicolons since it's by far the most common way to write JS and keeps subtle errors controlled.